ChIP-Seq/tutorials/TAL1_binding_site_identification.md

```diff
@@ -0,0 +1,420 @@
+To this end, ChIP-seq experiments were performed in multiple mouse cell types including G1E - a GATA-null immortalized cell line derived from targeted disruption of GATA-1 in mouse embryonic stem cells - and megakaryocytes.
+This dataset (GEO Accession: [GSE51338](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE51338)) consists of biological replicate Tal1 ChIP-seq and input control experiments.
+Input control experiments are used to identify and remove sampling bias, for example open/accessible chromatin or GC bias.
+
+Because of the long processing time for the large original files, we have downsampled the original raw data files to include only reads that align to chromosome 19 and a subset of interesting genomic loci pulled from the Wu et al. (2014) publication.
+
+**Table 1**: Metadata for ChIP-seq experiments in this tutorial. SE: single-end.
+
+| Cellular state | Datatype | ChIP Ab | Replicate | SRA Accession | Library type | Read length | Stranded? | DS.3 size (MB) |
+|---|---|:-:|:-:|---|:-:|:-:|:-:|---|
+| G1E | chIP-seq | input | 1 | SRR507859 | SE | 50 | No | 35.8 |
+| G1E | chIP-seq | input | 2 | SRR507860 | SE | 50 | No | 427.1 |
+| G1E | chIP-seq | Tal1 | 1 | SRR492444 | SE | 50 | No | 32.3 |
+| G1E | chIP-seq | Tal1 | 2 | SRR492445 | SE | 50 | No | 62.7 |
+| Megakaryocyte | chIP-seq | input | 1	| SRR492453 | SE | 50 | No | 57.2 |
+| Megakaryocyte | chIP-seq | input | 2 | SRR492454 | SE | 50 | No | 403.8 |
+| Megakaryocyte | chIP-seq | Tal1 | 1 | SRR549006 | SE | 50 | No | 340.3 |
+| Megakaryocyte | chIP-seq | Tal1 | 2 | SRR549007 | SE | 50 | No | 356.9 |
+
+<a name="example"/></a>
+# Analysis
+
+<a name="step1"/></a>
+### Step 1: Quality control
+
+As for any NGS data analysis, ChIP-seq data must be quality controlled before being aligned to a reference genome.
+
+:pencil2: ***Hands on!***
+
+1. Create and name a new history for this tutorial.
+
+2. Import the ChIP-seq raw data (\*.fastqsanger) from Zenodo [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.197100.svg)](https://doi.org/10.5281/zenodo.197100)
+
+
+    - In Zenodo: Right-click a FASTQ filename → Copy Link Address
+    - In Galaxy: Get Data → Upload File from your computer → Paste/Fetch data
+    - Paste the copied link into the dialog box and set the datatype to 'fastqsanger'
+    - Click Start
+    - Repeat for all FASTQ files
+
+    <img src="../images/upload_data_page.png"/>   
+    <figcaption><b>Figure X:</b> Data can be imported directly with links.</figcaption>
+    
+    
+    <img src="../images/data_uploaded.png"/>   
+    <figcaption><b>Figure X:</b> Imported datasets will appear in the history panel.</figcaption>
+    
+    
+3. In Galaxy, examine the data in a FASTQ file by clicking on the eye icon.
+
+    | :grey_question: Questions |
```

Comment:
We wanted to have questions "pop out" of the tutorial. Our goal is to force the user to stop and think critically about some of the steps they've just completed, rather than blindly run through all the clicking and parameter setting. We also want to keep the underlying biology at the forefront of the tutorial, and we felt that interspersing the tutorial with questions in a pop-up box was a good choice to do this. I am open to other formatting styles, but this is the best I could come up with without spending too much time messing around :)